curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/66c395a16c3ff33dc6f8e7c0.md

```diff
@@ -0,0 +1,133 @@
+---
+id: 66c395a16c3ff33dc6f8e7c0
+title: Step 26
+challengeType: 0
+dashedName: step-26
+---
+
+# --description--
+
+The `.sort()` method mutates the original array - in other words, it modifies the order of the elements directly. This is generally considered bad practise, as it can result in unexpected side effects.
```

Comment:
```suggestion
The `.sort()` method mutates the original array - in other words, it modifies the order of the elements directly. This is generally considered bad practice, as it can result in unexpected side effects.
```